src/watertap_contrib/seto/costing/units/med_tvc_surrogate.py

```diff
@@ -0,0 +1,258 @@
+        bounds=(0, None),
+        doc="Cost of labor per m3 distillate",
+    )
+
+    blk.cost_misc_per_vol_dist = pyo.Var(
+        initialize=0.033,
+        units=costing.base_currency / pyo.units.m**3,
+        bounds=(0, None),
+        doc="Cost of labor per m3 distillate",
+    )
+
+    blk.cost_disposal_per_vol_brine = pyo.Var(
+        initialize=0.02,
+        units=costing.base_currency / pyo.units.m**3,
+        bounds=(0, None),
+        doc="Cost of disposal per m3 brine",
+    )
+
+    blk.specific_energy_consumption_electric = pyo.Var(
+        initialize=1.5,
+        units=pyo.units.kWh / pyo.units.m**3,
+        bounds=(0, None),
+        doc="Specific electric energy consumption",
+    )
+
+    blk.hours_thermal_storage = pyo.Var(
+        initialize=0,
+        units=pyo.units.hr,
+        doc="Hours of thermal storage required",
+    )
+
+    blk.med_sys_A_coeff = pyo.Var(
+        initialize=6291,
+        units=pyo.units.dimensionless,
+        doc="LT-MED system specific capital A coeff",
+    )
+
+    blk.med_sys_B_coeff = pyo.Var(
+        initialize=-0.135,
+        units=pyo.units.dimensionless,
+        doc="LT-MED system specific capital B coeff",
+    )
+
+    blk.heat_exchanger_ref_area = pyo.Var(
+        initialize=302.01,
+        units=pyo.units.m**2 / (pyo.units.kg / pyo.units.s),
+        doc="Specific heat exchanger area from reference plant",
+    )
+
+    blk.heat_exchanger_exp = pyo.Var(
+        initialize=0.8,
+        units=pyo.units.dimensionless,
+        doc="Exponent for specific heat exchanger cost equation",
+    )
+
+    blk.fix_all_vars()
+
+
+@register_costing_parameter_block(
+    build_rule=build_med_tvc_surrogate_cost_param_block,
+    parameter_block_name="med_tvc_surrogate",
+)
+def cost_med_tvc_surrogate(blk):
+
+    med_tvc_params = blk.costing_package.med_tvc_surrogate
+    make_capital_cost_var(blk)
+    make_fixed_operating_cost_var(blk)
+
+    med_tvc = blk.unit_model
+    feed = med_tvc.feed_props[0]
+    dist = med_tvc.distillate_props[0]
+    brine = med_tvc.brine_props[0]
+    base_currency = blk.config.flowsheet_costing_block.base_currency
+
+    blk.total_system_cost = pyo.Var(
+        initialize=100,
+        units=base_currency,
+        doc="MED system cost",
+    )
+
+    blk.membrane_system_cost = pyo.Var(
+        initialize=100,
+        units=base_currency,
+        doc="Membrane system cost",
+    )
+
+    blk.evaporator_system_cost = pyo.Var(
+        initialize=100,
+        units=base_currency,
+        doc="Evaporator system cost",
+    )
+
+    blk.med_specific_cost = pyo.Var(
+        initialize=100,
+        units=pyo.units.USD_2018 / (pyo.units.m**3 / pyo.units.day),
+        doc="MED system cost per m3/day distillate",
+    )
+
+    blk.heat_exchanger_specific_area = pyo.Var(
+        initialize=100,
+        units=pyo.units.m**2 / (pyo.units.kg / pyo.units.s),
+        doc="Specific heat exchanger area",
```

Comment:
Why not have this created on the unit model?

Reply:
@Zhuoran29 I saw you noted that the changes were made, but can you address each comment? For example, this change wasn't made, and I suppose you have some thoughts on it.

Reply:
This variable is removed here and calculated in the unit model.